unittests/test_schedulers.py

```diff
@@ -445,11 +453,14 @@ def test_cancel(make_job, exec_ctx):
     minimal_job.wait()
     t_job = time.time() - t_job
     assert minimal_job.finished()
-    assert t_job < 30
 
     # Additional scheduler-specific checks
     sched_name = minimal_job.scheduler.registered_name
-    if sched_name in ('slurm', 'squeue'):
+
+    # This is a little over 30
+    assert t_job < 30
+
```

Comment:
I would move that back before the `sched_name` so that the `sched_name` is used close to its assignment. Also I don't understand vey much the comment. This generally should be less than 30s, unless the submit is super slow (which can happen occasionally when testing Slurm, but that generally should not be case).

Reply:
Oh that comment is wrong now (and I should have removed it!) I was testing before I had this actually implemented in flux and sort of taking notes. :laughing: